ingester/database.py

```diff
@@ -405,3 +405,13 @@ def get_result_query(self, query, parameters):
         with self.driver.session() as session:
             result = session.run(query, parameters).single()
         return result
+
+    def get_deployments(self, model_id):
+        query = """
+            MATCH (m:Model {model_id: $model_id})-[:HAS_DEPLOYMENT]->(d:Deployment)
+            RETURN properties(d) as deployment_info
+        """
+        with self.driver.session() as session:
+            result = session.run(query, model_id=model_id)
+            record = result.single()
```

Comment:
There's going to be more than one result.

Reply:
fixed.

<img width="700" alt="image" src="https://github.com/user-attachments/assets/1a4dc620-c5e7-45d4-94e5-0079717ebe91">